src/ui/components/CloseButton/CloseButton.jsx

```diff
@@ -0,0 +1,52 @@
+import { css } from '@emotion/react';
+import { theme } from '../../../shared/theme';
+import closeButtonUrl from '../../../public/img/closeButton.jpg';
+
+const CLOSE_BUTTON_BORDER_WIDTH = '10px';
```

Comment:
Chyba lepiej zmienić to na rem?

Reply:
W sumie to celowo bo chciałam żeby szerokość ramki się nie zmieniała, wtedy jak zmienia się root  font-size na 16px to byłaby bardzo wąska.

Reply:
Szerokość ramki faktycznie lepiej w px, zwłaszcza jak użytkownik zmieni sobie rozmiar fontu w przeglądarce 😀